doc/ref_model/chapters/chapter09.md

```diff
@@ -306,3 +318,137 @@ Topics include:
+
+This requires the collection of alarms and telemetry data from the physical layer (wires), the cloud infrastructure up to the network, application and services virtualized functions (VNF) running on top of Cloud Infrastructure, typically isolated by tenants.
+
+Long term trending data are essential for capacity planning purposes and typically collected, aggregated and kept over the full lifespan. To keep the amount of data collected manageable, automatic data reduction algorithms are typically used, e.g. by merging data points from the smallest intervals to more granular intervals.
+
+A cloud typically consists of one or more regional data centers, central offices, and edge sites. These are managed from redundant central management sites, each hosted in their own data centers.
+
+While many Telco Clouds start as a vertical cloud by hosting one prime application, e.g. IMS, the intent is to host more applications over time in order to truly maximize the cloud to its full potential. Therefore it is pure coincidence to have the same team responsible and manage the infrastructure and the application running on top. Network services and applications deployed on a Telco Cloud are managed by separate teams, within the same or different organizations and a monitoring solution must be capable of keeping the collection of monitoring data isolated between tenants and NFVI. At the same time, some monitoring data from the NFVI layer must selectively be available to tenant monitoring applications in order to correlate e.g. VNF metrics with the underlying infrastructure it currently runs on.
+
+<a name="9.6.1.1"></a>
+#### 9.6.1.1. What to observe
+
+Typically, when it comes to data collection, three questions arise:
+1. Which data to collect?
+2. Where to send the data?
+3. Which protocol/interface/format to use?
+
+<a name="9.6.1.2"></a>
+#### 9.6.1.2. What data to collect
+
+Assessment on which data to collect should start by iterating over the physical and virtual infrastructure components: 
+
+* Network Services across sites and tenants 							
+* Virtualized functions per site and tenant
+* Individual Virtual Machines and Containers
+* Virtualization infrastructure components
+* Physical servers (compute) and network elements
+* Toolservers with their applications (DNS, Identity Management, Zero Touch Provisioning, etc) 
+* Cabling 
+
+<a name="9.6.1.3"></a>
+#### 9.6.1.3. Data categories
+
+There are three main observability categories: metrics, alarms and logs:
+
+1. **Metrics** or telemetry report counters and gauge levels and can either be pulled periodically e.g. via SNMP or REST, or pushed as streams using gRPC, NETCONF, which receivers registered for certain sensors, or by registering as a publisher to a Kafka bus. These messages must be structured in order to get parsed successfully.
+
+2. **Alarms** indicate state changes, categorized by severity, often with a description of what just
```

Comment:
I feel the label Alarm here is trying to describe both the nature of how the observability could be done and the severity of that observation. I would rather categorize this group as Events and not Alarms. These Events are generally asynchronous and could in some cases have different levels of severity attached to them e.g. Alerts and Alarms if the wanted action from the Event consumer is known.

Reply:
Changed Alarms to Events